ibm/service/usagereports/resource_ibm_billing_report_snapshot.go

```diff
@@ -0,0 +1,482 @@
+// Copyright IBM Corp. 2023 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package usagereports
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/platform-services-go-sdk/usagereportsv4"
+)
+
+func ResourceIBMBillingReportSnapshot() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIBMBillingReportSnapshotCreate,
+		ReadContext:   resourceIBMBillingReportSnapshotRead,
+		UpdateContext: resourceIBMBillingReportSnapshotUpdate,
+		DeleteContext: resourceIBMBillingReportSnapshotDelete,
+		Importer:      &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": &schema.Schema{
```

Comment:
Acccount id can be taken internally in the code.
Do we really need user to pass accountid?
Same applies to datasource as well.

Reply:
Incorporated the change to pick account id internally.